gcloud/bigquery/test_table.py

```diff
@@ -216,6 +216,23 @@ def test_ctor_w_schema(self):
                               schema=[full_name, age])
         self.assertEqual(table.schema, [full_name, age])
 
+    def test_query_id_property(self):
+        client = _Client(self.PROJECT)
+        dataset = _Dataset(client)
+        table = self._makeOne(self.TABLE_NAME, dataset)
+        expected_result = ''.join([self.PROJECT, ':', self.DS_NAME,
+                                   '.', self.TABLE_NAME])
```

Comment:
Sure thing. I was trying to avoid accidentally wrong string formatting in Py3 since

```b
Python 3.4.3 (default, Oct 14 2015, 20:28:29) 
>>> '%s.%s' % ('foo', 'bar')
'foo.bar'
>>> '%s.%s' % (b'foo', b'bar')
"b'foo'.b'bar'"
```